### ISSUE TITLE
Add an automated a11y standards checker tool

### ISSUE THREAD

**Issue by jroebu14** [bbc/simorgh#9240]
**Is your feature request related to a problem? Please describe.**
We recently decided to remove use of [BBC Accessibility Standards Checker](https://github.com/bbc/bbc-a11y) because it is no longer actively supported and doesn't receive patches or updates.

We need another accessibility standards checker to take its place to ensure our service is as accessible as it can be.

**Describe the solution you'd like**
We would like to use the Axe core accessibility testing engine to run against our pages in the CI pipeline.

We need to have a discussion or investigation around the most suitable tool for rendering pages for Axe core to run tests against e.g. Puppeteer, Cypress.

This should run against the A, AA and best practice rules.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**
web-core implemented a solution in PR#4423
